std/random.d

```diff
@@ -1183,6 +1183,130 @@ A singleton instance of the default random number generator
 }
 
 /**
+A random bool generator that has the underlying random integral
+generation batched.
+ */
+struct RandomBool(Rng = Random)
+    if (isUniformRNG!Rng || isUniformRNG!(typeof(*Rng.init)))
+{
+    private Rng parent;
+    private uint mask;
+    private uint accumulator;
+
+    /// Always `false`
+    enum bool empty = false;
+
+    static if (isPointer!Rng)
+    {
+        this(Rng rng) @safe
+        {
+            parent = rng;
+            initPrivates();
+        }
+    }
+    else
+    {
+        /**
```

Comment:
I think you need to put the doc comment in the first branch of the static if in order for it to be visible to ddoc.